settings.py

```diff
@@ -918,7 +918,7 @@
 DAYS_TO_KEEP_DEVICE_LOGS = 60
 
 MAX_RULE_UPDATES_IN_ONE_RUN = 10000
-
+SKIP_GOOGLE_ANALYTICS = False
```

Comment:
This isn't skipping google analytics. it's only skipping google tag manager. and we want google analytics on eniksay/icds anyways